core/templates/dev/head/services/stateful/FocusManagerService.ts

```diff
@@ -20,6 +20,7 @@
  * page.
  */
 
+require('services/IdGenerationService.ts');
```

Comment:
What about `DeviceInfoService` and `LABEL_FOR_CLEARING_FOCUS`? Also add empty line after this line.

Reply:
`LABEL_FOR_CLEARING_FOCUS` is in app.constants.
I've added a require for `DeviceInfoService`.

Reply:
Ordering. (files that are more "deep" in the directory structure should be last)